### PR TITLE
[AKS] change default node size to Standard_DS1_v2

### PR DESCRIPTION
**Pull request by mboersma**
Changes the default Kubernetes VM node size for `az aks create` to `Standard_DS1_v2`.

cc: @sauryadas @slack 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [X] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [X] Each command and parameter has a meaningful description.
- [X] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by promptws** [comment]
View a preview at https://prompt.ws/r/Azure/azure-cli/5407
This is an **experimental preview** for @microsoft users.

File: src/command_modules/azure-cli-acs/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.0.26
+++++++
+* `az aks create` defaults to Standard_DS1_v2 sizes for VM nodes
```
Comment (tjprescott) [inline src/command_modules/azure-cli-acs/HISTORY.rst]: Recommend rewording this 

* `aks create` VM node size default changed from "Standard_D1_v2" to "Standard_DS1_v2".

**Comment by derekbekoe** [comment]
@mboersma We are looking into the `Dependency compatibility errors found!` error.
One of our dependencies `adal` released an update today so there's a dependency mismatch affecting all PRs.

**Comment by mboersma** [comment]
No worries--let me know if I need to rebase this at some point to fix things.

**Pull request merged**